### PR TITLE
High Energy Lab Improvements

### PR DESCRIPTION
**Pull request by AtnerNT**
NewHIECASprites

### Changelog:

CL: [Improvement] Remapped Squarestation's high energy lab to be better integrated into the rest of the science department.
CL: [Improvement] The H.I.E.C.A has received new sprites, improving consistency in the form between rotations.
CL: [Improvement] Minor sprite change to the down sprite of the R&D Laser Projector
CL: [Balance] Adjusted various research values on varying items.
CL: [Balance] Reduced the number of H.I.E.C.A's on Fallstation and Ministation from 3 to 2, as to warrant the need to use reflectors.
CL: [Balance] Added an item blacklist for what can be put inside the research item plinth. This is used to blacklist high risk items such as the captain's hat and chain of command to prevent them being destroyed.
CL: [Balance] Foam reaction has been altered to match the TG reaction of Fluorosurfactant + Water instead of just Fluro surfactant
CL: [Balance] Several ordnance bounties have had their reagents/reactants changed to make more sense (I.E To detect the foam reaction opposed to foaming agent that does not form foam)
CL: [Balance] Amount of Ordnance bounties increased from 10 to 15
CL: [Fix] Fixed not being able to remove items from research item plinth while connected to a server.

### CODE REVIEW

**Comment by MaxIsJoe** [comment]
idk how to feel about the new sprites, i kinda like the old ones

**Comment by AtnerNT** [comment]
> idk how to feel about the new sprites, i kinda like the old ones

image
For a comparison here are the old ones, the left and right have been changed, as it didnt reall make sense in 3 dimensions and I felt the perspective was weird

File: UnityProject/Assets/Scripts/Systems/Research/LaserPuzzle/ItemPlinth.cs [hunk 1]
```diff
@@ -41,32 +54,30 @@ public bool WillInteract(PositionalHandApply interaction, NetworkSide side)
...
+			OnItemChange?.Invoke();					
 		}
 		else
 		{
-			DisplayedItem = interaction.HandSlot.Item;
-			Inventory.ServerTransfer(interaction.HandSlot, itemStorage.GetNextEmptySlot());
-			OnItemChange?.Invoke();
+			//if (interaction.HandSlot.Item == null) return;
+
+			if (blackListedItemNames.Contains(interaction.HandObject.Item().InitialName)) Chat.AddExamineMsg(interaction.Performer, $"The light on the pinth blinks red, refusing storage of the {interaction.HandObject.ExpensiveName()}");
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Systems/Research/LaserPuzzle/ItemPlinth.cs]: put the body of the if in its own line with brackets

**Comment by Bod9001** [comment]
"CL: [New] Squarestation has received a high energy lab." technically it had gone but it was in space

**Comment by AtnerNT** [comment]
> "CL: [New] Squarestation has received a high energy lab." technically it had gone but it was in space

Oop forgot to remove the old one from the map so thats a good reminder. ill rephrase the changelog

**Comment by AtnerNT** [comment]
That should be suggestions applied and ready to merge if tests pass

**Review by MaxIsJoe — APPROVED** [review]

**Pull request merged**